src/styles/content.scss

```diff
@@ -57,9 +60,177 @@ table.os-raise-standardtable {
   }
 }
 
+%os-raise-basictable {
+  /* border-collapse: separate paired with border-spacing: 0 allows for a border-radius
+     to be applied to the entire table (border-collapse: collapse will ignore any border
+     radius applied to the table)
+  */
+  border: 2px solid $raise-table-color;
+  border-collapse: separate;
+  border-spacing: 0;
+  border-radius: 5px;
+
+  caption {
+    color: $raise-light-black;
+    font-size: 1.6rem;
+    font-weight: 600;
+    text-align: left;
+  }
+
+  thead {
+    background-color: $raise-table-color;
+  }
+
+  th,
+  td {
+    text-align: center;
+    padding: 16px;
+  }
+
+  th {
+    font-size: 1.2rem;
+    font-weight: 700;
+    color: #fff;
+  }
+
+  td {
+    font-size: 1.067rem;
+    color: $raise-light-black;
+    background-color: #fff;
+    border-left: 1px solid $raise-table-color;
+    border-top: 1px solid $raise-table-color;
+  }
+}
+
+table.os-raise-doubleheadertable {
+  table-layout: fixed;
+  width: 100%;
+
+  caption {
+    color: $raise-light-black;
+    font-size: 1.6rem;
+    font-weight: 600;
+    text-align: left;
+  }
+
+  th,
+  td {
+    border: 1px solid $raise-light-gray;
+    text-align: center;
+    padding: 16px;
+  }
+
+  th {
+    color: #000;
+    font-size: 1.2rem;
+    background-color: #84ada8;
+    font-weight: 700;
+  }
+
+  thead tr th:first-child {
+    border: none;
+    background-color: #fff;
+  }
+
+  td {
+    color: $raise-light-black;
+    font-size: 1.067rem;
+    background-color: #fff;
+  }
+}
+
+table.os-raise-textheavytable {
+  @extend %os-raise-basictable;
+
+  table-layout: fixed;
+  width: 100%;
+
+  /* stylelint-disable-next-line no-descending-specificity */
+  th,
+  td {
+    text-align: left;
+  }
+
+  td:first-child {
+    border-left: none;
+  }
+
+  p {
+    margin-bottom: 0;
+  }
```

Comment:
I'm not sure if this is a good idea as it will break something like:

```html
      <td>
        <p>A11(B) simplfy numeric and algebraic expressions using the laws of exponents, including integral and rational exponents</p>
        <p>A11(B) simplfy numeric and algebraic expressions using the laws of exponents, including integral and rational exponents</p>
      </td>
```

Reply:
Agreed - the reasoning behind this was to try and align the design to what Maddy provided. I can revert back. Below is the difference in spacing w/ and w/o the margin-bottom.

Without margin-bottom:
![image](https://github.com/openstax/k12-apps-raise/assets/37000585/82d6dbc9-c2f5-4f8b-bad2-8a4e0f77fa27)


With margin-bottom (default 1rem):
![image](https://github.com/openstax/k12-apps-raise/assets/37000585/2967c3c0-809d-4b8b-bf2e-279659a3200b)

Reply:
:thinking: -- I suppose in this specific type of table one could do something like the following to avoid the margin added by `<p>`:

```html
<table class="os-raise-textheavytable">
  <caption> Texas Essential Knowledge and Skills (TEKS)
  <thead>
    <tr>
      <th scope="col">TEKS</th>
      <th scope="col">Explination of Coverage</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>
        A11(B) simplfy numeric and algebraic expressions using the laws of exponents, including integral and rational exponents
      </td>
      <td>
        Partial coverage: Lesson provides content that covers part of this TEKS. The parts that are covered have been underlined
      </td>
    </tr>
  </tbody>
</table>
```

But, that doesn't work so well when you do want `<p>`s in the cell. Perhaps the general case warrants creating a utility class like `os-raise-mb-0` :man_shrugging: ?

Reply:
:+1: -- Yeah, that's what I'm thinking.